Modules/HR/Http/Controllers/EmployeeController.php

```diff
@@ -102,7 +102,9 @@ public function reviewDetails(Employee $employee)
         $assessments = Assessment::where('reviewee_id', $employee->id)
             ->orderBy('created_at', 'desc')
             ->get();
-        $employees = Employee::all();
+        $employees = Employee::whereHas('user', function ($query) {
+            $query->whereNull('deleted_at');
+        })->get();
```

Comment:
I think, can we write like this?
```suggestion
        $employees = Employee::whereNull('deleted_at')->get();
```